rackcity/api/serializers/asset_serializers.py

```diff
@@ -530,11 +531,43 @@ def get_neighbor_assets(hostname, id, nodes, edges, change_plan=None):
                 edges.append(
                     {"from": source_port.asset.id, "to": destination_port_asset.id}
                 )
+        nodes, edges = add_blades_to_graph(id, nodes, edges, change_plan)
+        nodes, edges = add_chassis_to_graph(id, nodes, edges, change_plan)
         return nodes, edges
     except ObjectDoesNotExist:
         return
 
 
+def add_blades_to_graph(id, nodes, edges, change_plan=None):
+    if change_plan:
+        blades = AssetCP.objects.filter(chassis=id, change_plan=change_plan)
+    else:
+        blades = Asset.objects.filter(chassis=id)
```

Comment:
even if youre on a change plan, you still need to get the blades for a chassis that might not be on the change plan.